kit-contents-checklist/kit-contents-checklist.tex

```diff
@@ -0,0 +1,72 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{tabularx}
+\setlength{\parindent}{0cm}
+
+\pagenumbering{gobble}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Student Robotics Kit Contents List}}
+
+    \medskip
+
+\end{center}
+
+\textbf{TLA:} \hrulefill
+
+\medskip
+
+Each team is responsible for ensuring that they return these items from their kit. This list is also available in the appendices of the rulebook.
+
+\begin{center}
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|r|c|X|}
+\hline
+    \textbf{Part} & \textbf{Quantity} & \textbf{Returned?} \\
+\hline
+ 18l Really Useful Box & 1 & \\ \hline
+
+ %% SR-specific stuff
+ Power Board & 1 & \\ \hline
+ Brain Board (Odroid U3) & 1 & \\ \hline
+ Motor Board & 2 & \\ \hline
+ Servo Board & 1 & \\ \hline
+ Ruggeduino & 1 & \\ \hline
+ Screw Shield & 2 & \\ \hline
+
+ %% USB stuff
+ USB Hub & 2 & \\ \hline
+ USB Memory Stick & 1 & \\ \hline
+ USB WiFi Adapter & 2 & \\ \hline
+ Webcam (C500 or C270) & 1 & \\ \hline
+ USB A to USB B lead & 3 & \\ \hline
+ USB A to USB Micro-B lead & 5 & \\ \hline
+
+ %% Battery stuff
+ Lithium Polymer Battery & 2 & \\ \hline
+ Battery Charger (IMAX B6 or HobbyKing E4) & 1 & \\ \hline
+ Charger Power Supply and Mains Cable & 1 & \\ \hline
+ Battery charging bag & 1 & \\ \hline
```

Comment:
maybe put the bag next to the batteries, since the batteries will likely be in the bag?

Reply:
I just pulled this list as-is from the rules